sklearn/feature_selection/tests/test_feature_select.py

```diff
@@ -404,7 +404,7 @@ def single_fdr(alpha, n_informative, random_state):
             # FDR = E(FP / (TP + FP)) <= alpha
             false_discovery_rate = np.mean([single_fdr(alpha, n_informative,
                                                        random_state) for
-                                            random_state in range(30)])
+                                            random_state in range(100)])
```

Comment:
# FDR = E(FP / (TP + FP)) <= alpha
FDR is count by average many times (N) of "FP/(TP+FP)"
The larger of N, FDR is more accuracy.  
Actually,  N is the larger the better here.